packages/react-scripts/template/src/serviceWorker.js

```diff
@@ -63,8 +63,8 @@ function registerValidSW(swUrl, config) {
         if (installingWorker == null) {
           return;
         }
-        installingWorker.onstatechange = () => {
-          if (installingWorker.state === 'installed') {
+        installingWorker.onstatechange = stateChangeEvent => {
+          if (stateChangeEvent.target.state === 'installed') {
```

Comment:
`target` or `currentTarget`?

Reply:
I was able to get it working using `target`. If you believe this should be `currentTarget` I would happy to change and test.

Reply:
After some more testing, I was able to get this working without the changes I made. I have reverted the changes.